apps/server/src/modules/h5p-editor/controller/dto/h5p-editor.params.ts

```diff
@@ -1,11 +1,13 @@
 import { IContentMetadata } from '@lumieducation/h5p-server';
-import { ApiProperty } from '@nestjs/swagger';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { SanitizeHtml } from '@shared/controller';
 import { IsMongoId, IsNotEmpty, IsObject, IsOptional, IsString, Matches } from 'class-validator';
 
 export class GetH5PContentParams {
-	@ApiProperty()
+	@ApiPropertyOptional()
 	@Matches('([a-z]+-[a-z]+)')
 	@IsString()
+	@SanitizeHtml()
```

Comment:
Why do you need @SanitizeHtml hier? You have @Matches or?

Reply:
@virgilchiriac said to use that decorator for every string
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4190#discussion_r1238500255

Reply:
Your regex in @matches allows only a-z, an ID cannot contain html tags